tests/unit/test_config.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2023 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+""" test_config.py is part of csp-billing-adapter and provides units tests
+for the Config class.
+"""
+from pytest import raises
+from yaml.parser import ParserError
+
+from csp_billing_adapter.config import Config
+from csp_billing_adapter.adapter import get_plugin_manager
+
+good_config_file = 'tests/data/config_good.yaml'
```

Comment:
With a data subdir in tests it should probably be skipped in pytest and flake8:

setup.cfg
```
[tool:pytest]
norecursedirs = data
testpaths = tests

[flake8]
exclude = data
```